humanoid_league_team_communication/humanoid_league_team_communication/communication.py

```diff
@@ -0,0 +1,53 @@
+import socket
+
+from rclpy.node import Node
+
+
+class SocketCommunication:
+
+    def __init__(self, node: Node, logger, team_id, robot_id):
+        self.logger = logger
+
+        self.socket: socket.socket = None
+        self.target_host = node.get_parameter('target_host').get_parameter_value().string_value
+
+        if self.target_host == '127.0.0.1':
```

Comment:
In theory it would work I think, but so far, we have only been using IP addresses.
I'll change it to actually use a `target_ip` as a config value and use the [ipaddress module](https://docs.python.org/3/library/ipaddress.html)